### ISSUE TITLE
Excessive space between `r` and `s` when whitespace follows?

### ISSUE THREAD

**Issue by nmsmith** [eigilnikolajsen/commit-mono#107]
Is it just me, or do the trailing `s`'s look detached from their word ("numbers") in the below image?

<img width="224" alt="Screenshot 2024-05-10 at 6 01 47 pm" src="https://github.com/eigilnikolajsen/commit-mono/assets/909604/da21820c-0df5-4974-a520-eb596dbc25c8">

**Note:** This is with smart kerning turned on.

**Comment by eigilnikolajsen** [comment]
Good point, I'll have a look. Maybe `r` needs to moved more to the right. It is impossible to get correct spacing everywhere though, since some characters are narrow (like r, i, l, etc.)
